samples/loadimpactair_script_sample.js

```diff
@@ -0,0 +1,403 @@
+}
+
+/**
+ * Returns __EVENTVALIDATION found in body or empty string if not found
+ */
+function findEventvalidation(body) {
+	var strEventvalidation = "";
+    var findEV = new RegExp('id="__EVENTVALIDATION" value="(.*?)"' )
+	var reMatch = findEV.exec(body);
+	if (reMatch != null) {
+    	strEventvalidation = reMatch[1];
+	}
+	else {
+		console.log("No eventvalidation")
+	}
+
+	return strEventvalidation;
+}
+
+
+/**
+ * Returns __VIEWSTATE found in body of Ajax call return (i.e. panel) or empty string if not found
+ */
+function findPanelViewstate(body) {
+	var strViewstate = "";
+	var findVS = new RegExp('|__VIEWSTATE|(.*?)|');
+	var reMatch = findVS.exec(body);
+	if (reMatch != null)
+	{
+		strViewstate = reMatch[1];
+	}
+	else
+	{
+		console.log("No panel viewstate");
+		console.log(body);
+	}
+}
+
+
+/**
+ * Returns __EVENTVALIDATION found in body of Ajax call return (i.e. panel) or empty string if not found
+ */
+function findPanelEventvalidation(body) {
+	var strEventvalidation = "";
+	var findEV = new RegExp('|__EVENTVALIDATION|(.*?)|');
+	var reMatch = findEV.exec(body);
+	if (reMatch != null)
+	{
+		strEventvalidation = reMatch[1];
+	}
+	else
+	{
+		console.log("No panel eventvalidation");
+		console.log(body);
+	}
+}
+
+
+
+/**
+ * Set k6 test execution options
+ * vusMax: Maximum number of vus that can be executed in a test
+ * duration: the duration of the test run
+ * maxRedirects: the number redirects allowed for a request before it is considered failed
+ * insecureSkipTLSVerify: true will allow all certificates, dev and homemade as well
+ * stages: from 0 to 10 vu in 20 seconds
+ * 		   from 10 to 15 vu in 20 seconds
+ * 		   from 15 to 20 vu in 20 seconds
+ * 		   stay at 20 vu for 40 seconds
+ * 		   from 20 to 80 vu in 20 seconds
+ * 		   stay at 80 vu for 180 seconds
+ * 	  	   from 80 to 0 vu in 10 seconds
+ */
+export let options =  {
+	vusMax: 100,
+	duration: "10m",
+	stages: 
+		[{target: 10, duration: "20s"},
+		 {target: 15, duration: "20s"},
+		 {target: 20, duration: "20s"},
+		 {target: 20, duration: "40s"},
+		 {target: 80, duration: "20s"},
+		 {target: 80, duration: "180s"},
+		 {target: 0, duration: "10s"}],	
+	maxRedirects: 15,
+	insecureSkipTLSVerify: true,
+}
+
+
+// Create new trend metric named "group"
+var tr = new Trend("group");
+
+// Use a top-level function wrapper to allow us to return from it. If you don't need to
+// return from your script at any point, you can skip the wrapper - VU scopes are isolated.
+export default function() {
+
+	// Something to hold the responses in
+	var res = null;
+	// Timer start time
+	var tStart = 0;
+	// Timer end time
+	var tEnd = 0;
+
+	// First step - go to the landing page
+	// Put in group and name group something resaonable
+	// Batches are from how the real page is actually loaded
+	//   first the page then the next level content
+	//   and the last batch is the content from the bottom level of the content hierarchy
+	group ("01_landingpage", function () {

```

Comment:
See previous comment - it's tied to the actual user journey step you have and how to analyse results. It's actually both.